libs/ui/elements/src/lib/search-results-error/search-results-error.component.html

```diff
@@ -13,14 +16,29 @@
     </div>
     <div translate>search.error.couldNotReachApi</div>
   </div>
-  <div *ngIf="type === types.RECEIVED_ERROR">
+  <div *ngIf="type === types.NO_LINK" data-test="no-link-error">
+    <div class="relative opacity-40">
+      <mat-icon class="material-symbols-outlined face">computer</mat-icon>
+      <mat-icon class="material-symbols-outlined question-mark1"
+        >question_mark</mat-icon
+      >
+      <mat-icon class="material-symbols-outlined question-mark2"
```

Comment:
What are these two question marks icons for ? They are not in the screenshot you provided.

Reply:
Wrong copy/pasta. They don't even display 😆 
Fixed it. Good catch !